### PR TITLE
remove app

### PR DESCRIPTION
**Pull request by dfulu**
# Pull Request

## Description

Remove the app from PVNet now that it is included in the [pvnet_app repo](https://github.com/openclimatefix/pvnet_app)

- Remove app
- Remove tests
- Remove unneeded requirements
- Remove docker build

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,5 @@
-nowcasting_utils
-PVNet-summation>=0.0.7
-nowcasting_datamodel==1.5.10
 ocf_datapipes==1.2.44
+nowcasting_utils
```
Comment (peterdudfield) [inline requirements.txt:2]: be nice to to include this? whats it needed for?

**Review by peterdudfield — APPROVED** [review]
looks good, one small comment

File: requirements.txt (hunk 1 above)
Comment (dfulu) [reply requirements.txt:2]: Its for some of the metrics. The exponential weighted loss and the forecast mae at each time horizon. I could pull those out and put them in PVNet if we want to remove the import?

Comment (peterdudfield) [reply requirements.txt:2]: lhmm, thats ok, leave it

**Pull request merged**